Makefile

```diff
@@ -1044,6 +1044,13 @@ u-boot-pistachio-nor.img: u-boot-dtb.img u-boot-spl-pistachio.bimg FORCE
 	@dd if=u-boot-spl-pistachio.bimg of=$@ bs=4K count=128 conv=notrunc
 	@dd if=u-boot-dtb.img of=$@ bs=4K count=256 seek=128 conv=notrunc
 
+ifeq ($(UBOOTRELEASE),)
+	-mv  u-boot-pistachio-nor.img u-boot-$(subst $\",,$(CONFIG_SYS_CONFIG_NAME))-$(UBOOTVERSION).img
+else
+	-mv u-boot-pistachio-nor.img "u-boot-$(subst $\",,$(CONFIG_SYS_CONFIG_NAME))-\
```

Comment:
Need to change JenkinsFile too .. It has still u-boot-pistachio-nor.img that's why it has failed
http://jenkins.creatordev.io/job/CreatorDev/job/u-boot/job/PR-29/lastBuild/console